src/corefx/System.Globalization.Native/pal_collation.c

```diff
@@ -184,19 +157,18 @@ static UCharList* GetCustomRules(int32_t options, UColAttributeValue strength, i
     }
 
     // If we need to create customRules, the KanaType custom rule will be 88 kana characters * 4 = 352 chars long
```

Comment:
The globalization constants like this are changing all the time. I am not sure whether it is a good idea to hardcode them into the code.

cc @tarekgh

Reply:
I am not expecting this will change regularly as Unicode not changing the half width much. So, I think we are ok to hard code the values here. Otherwise, we'll have to expose some property in our Unicode Category class to return the value matches the Unicode version used there 

---
In reply to: [277029562](https://github.com/dotnet/coreclr/pull/24100#discussion_r277029562) [](ancestors = 277029562)

Reply:
The values are defined and used within one function, so if anything changes it's still within the same function scope.